README.md

```diff
@@ -4,30 +4,47 @@
 
 Travis: [![Build Status](https://travis-ci.org/intermine/bluegenes.svg?branch=dev)](https://travis-ci.org/intermine/bluegenes) GitLab: [![pipeline status](https://gitlab.com/yochannah/bluegenes/badges/dev/pipeline.svg)](https://gitlab.com/yochannah/bluegenes/commits/dev)
 
+_An InterMine Web-based GUI_
+
+
 ## About
+
 BlueGenes is designed to make searching and analysing genomic data easy. It's powered by [InterMine](http://intermine.org/) web services, meaning that the data from over 30 InterMines worldwide can be accessed from the same familiar interface.
 
-[Try BlueGenes now](http://bluegenes.apps.intermine.org/)
+### Screenshots
+
+![BlueGenes screenshots](http://i.imgur.com/zwp0uxM.jpg)
+
+
+## Try Online
 
+[BlueGenes Live](http://bluegenes.apps.intermine.org/)
 
-![BlueGene screenshots](http://i.imgur.com/zwp0uxM.jpg)
 
-**Feedback:** Please create an issue in this repo or email `info - at - intermine - dot - org`
+## Contributing
 
-## Setting up your dev environment and running BlueGenes
+Check out our [contributing guidelines](https://github.com/intermine/bluegenes/blob/dev/CONTRIBUTING.md).
 
-- **Development BlueGenes** If you want to run BlueGenes locally so that you can modify it, please see [docs/getting-started](docs/getting-started.md).
-- **BlueGenes as built into InterMine 2.0** If you have upgraded to InterMine 2.0 (the Gradle release), see the [instructions to launch a BlueGenes target](https://intermine.readthedocs.io/en/intermine-2.0/system-requirements/software/gradle/index.html#deploy-blue-genes). 
-- **Standalone BlueGenes** If you'd like to launch BlueGenes without it being associated directly with a single InterMine, or if you haven't upgraded to InterMine 2.0 yet (but you're above 1.8), please see [docs/production-builds](docs/production-builds.md).
 
-In all cases, you may also wish to see the [docs](docs/README.md) and [BlueGenes config instructions](docs/configuring-bluegenes.md)
+### Setting up a local development environment
 
-### Contributing to BlueGenes
+[Configuring a local build](docs/configuring.md)
 
-Wow, you're awesome! Check out our [contributing guidelines](https://github.com/intermine/bluegenes/blob/dev/CONTRIBUTING.md)
```

Comment:
Small note - I try to intentially cultivate a friendly and encouraging tone in the readme rather than a bland, professional one. :) not a big thing though.